### DISCUSSION TITLE
Quadlet and configmap re-usability

### DISCUSSION THREAD

**Discussion by odra** [containers/podman#19555] [Q&A] [answered]
Hello,

Is it possible to "re-use" Configmaps in Quadlet?

One can use `podman play kube --configmap ...` when invoking the command directly, is it possible to do the same in a `.kube` file?

**Comment by rhatdan** [comment, marked as answer]
man quadlet
```
   ConfigMap=
       Pass  the Kubernetes ConfigMap YAML at path to podman kube play via the
       --configmap argument.  Unlike the configmap  argument,  the  value  may
       contain  only  one path but it may be absolute or relative to the loca‐
       tion of the unit file.
```

**Comment by odra** [comment]
I totally missed that, I guess I need another cup of coffee :man_facepalming:
